tests/test_adapter_dataclasses.py

```diff
@@ -43,10 +43,13 @@ def test_module_import_error(self):
             from itemadapter.adapter import DataclassAdapter
 
             self.assertFalse(DataclassAdapter.is_item(DataClassItem(name="asdf", value=1234)))
-            with self.assertRaises(RuntimeError, msg="attr module is not available"):
+            with self.assertRaises(RuntimeError, msg="dataclasses module is not available"):
```

Comment:
I'm not sure why it's been passing. Probably, assertRaises doesn't work as intended here.

Reply:
I believe that is the message that shows if the assert fails. We can use `assertRaisesRegex` if the goal is for the assert of fail if the exception message does not match this one.